src/features/user/TreeMapper/Analytics/components/SpeciesPlanted/index.tsx

```diff
@@ -239,7 +239,7 @@ export const SpeciesPlanted = () => {
       });
     }, 2000);
 
-    if (process.env.ENABLE_ANALYTICS === 'true' && project) {
+    if (process.env.ENABLE_ANALYTICS && project) {
       fetchPlantedSpecies();
```

Comment:
I feel we should explicitly check ENABLE_ANALYTICS for a specific value and otherwise disable it. In the current implementation, any provided value will enable analytics.

Reply:
@Shreyaschorge

Reply:
NA - ENABLE_ANALYTICS is a boolean value